### ISSUE TITLE
Errors if target file is missing

### ISSUE THREAD

**Issue by ricklecoat** [erikreagan/er.cache_buster.ee_addon#3]
The plugin throws PHP errors when the file that cache_buster references is missing. When logged in as a SuperAdmin these errors prevent the page from rendering. The errors reference line 52 of the plugin:

```
Severity: Warning
Message: filemtime() [function.filemtime]: stat failed for /Users/Rick/Sites/above_root/public_html/library/scripts/someScript.min.js
Filename: cache_buster/pi.cache_buster.php
Line Number: 52
```

Perhaps the plugin could check that the file exists before it does it's timestamp thing, or maybe just limit the errors to the template debugger (Derek Jones’ suggestion). Thanks.

**Comment by erikreagan** [comment]
Thanks for the heads up. I'll tweak the logic a bit to prevent those errors.

Erik

**Comment by ricklecoat** [comment]
Awesome. Thanks Eric. Any chance of a heads-up once you've done that? It'd serve as a handy reminder to myself to download and apply the update, rather than checking Devot-ee repeatedly and then forgetting. Either way, many thanks. 

Cache_buster is incredibly useful, by the way -- I never used EE's templates for cs or js, so the whole cache-busting thing was a bit of a faff, but that single plug-in means I never again have to tell a client to empty their browser cache (well, not for css or js, anyway). A default inclusion on every project for me.

Warmest regards;
Rick

## 

SHARK ATTACK LIMITED: A Design Studio
+44 20 333 20200

http://www.sharkattack.co.uk
http://www.linkedin.com/in/ricklecoat
http://twitter.com/sharkattackdsgn

---

6 Burroughs Gardens, London, NW4 4AU
Registered in England & Wales, No. 3886438
VAT No. 982 2091 15

**Comment by erikreagan** [comment]
> Any chance of a heads-up once you've done that? It'd serve as a handy reminder to myself to download and apply the update, rather than checking Devot-ee repeatedly and then forgetting.

Absolutely. I'll post the update here in this Issue which I believe will notify you via email.

**Comment by ricklecoat** [comment]
Fab. Thanks again Erik.
Rick

## 

SHARK ATTACK LIMITED: A Design Studio
+44 20 333 20200

http://www.sharkattack.co.uk
http://www.linkedin.com/in/ricklecoat
http://twitter.com/sharkattackdsgn

---

6 Burroughs Gardens, London, NW4 4AU
Registered in England & Wales, No. 3886438
VAT No. 982 2091 15

**Comment by ricklecoat** [comment]
Hi Erik, did this ever get updated? (And will the plugin as it currently stands work with EE3?)

**Comment by ricklecoat** [comment]
Just a nudge on my last comment (from October 2015). Are there any plans to get this working with EE3? Or is it EE2-only now? (I'm really hoping for the former, since I've taken to using it in every project to serve up my flat .css and .js files).
